core/tests/protractor_utils/ExplorationPlayerPage.js

```diff
@@ -197,31 +204,35 @@ var ExplorationPlayerPage = function() {
       conversationInput, answerData);
     var waitingForResponseElem = element(by.css(
       '[ng-if="!data.oppiaResponse && isCurrentCardAtEndOfTranscript()"]'));
```

Comment:
Would you want a protractor tag for this? This div can be found in input_response_pair_directive.html at line 24( the thinking dots).

Reply:
Oh gosh, yes. Done.